sleap_roots_training/train.py

```
@@ -0,0 +1,412 @@
+        Args:
+            csv_path (Path): Path to the CSV file containing train-test splits paths.
+            use_existing_model (bool, optional): Whether to use an existing model for evaluation.
+        """
+        df = load_training_data(csv_path)
+        grouped = get_training_groups(df)
+
+        for version, group in grouped:
+            print(f"Processing version {version}...")
+            print(f"Group: {group}")
+            process_training(
+                project_name=PROJECT_NAME,
```

Ruff (0.8.2)

Undefined name `PROJECT_NAME`

(F821)